packages/jest-config/src/normalize.js

```diff
@@ -682,6 +682,7 @@ export default function normalize(options: InitialOptions, argv: Argv) {
       case 'watch':
       case 'watchAll':
       case 'watchman':
+      case 'i':
```

Comment:
I think this should state `runInBand`, `i` is an alias

Reply:
thanks 
as you can see when i did my code cleanup => i didn't retest it 
_(working with a baby with one hand and the other one in the keyboard ^^)_
I was also missing a file for my fix (added)

I'll add tests wanted to have a first feedback.